lib/order.rb

```diff
@@ -0,0 +1,93 @@
+require_relative 'menu'
+require_relative 'confirmation'
+
+class Order
```

Comment:
This class is doing a lot of things! Each method is concisely written, but there's something of an array of responsibilities that could potentially be grouped further.

If you were to group these methods together following this technique, which further classes would you consider drawing out? https://github.com/makersacademy/skills-workshops/blob/main/practicals/object_oriented_design/encapsulation.md